code/core/src/manager/components/sidebar/Tree.tsx

```diff
@@ -120,12 +117,12 @@ export const LeafNodeStyleWrapper = styled.div(({ theme }) => ({
 
   '&[data-selected="true"]': {
     color: theme.color.lightest,
-    background: theme.color.secondary,
+    background: theme.base === 'dark' ? darken(0.18, theme.color.secondary) : theme.color.secondary,
```

Comment:
Judging from your design intent here, my suggestion on Select Option is probably not right. Should we always apply the pattern of using darken only in light themes, and the original color in dark themes?

Reply:
No. This is a funky one along with button where we actually want to use a darker color than the dark theme's secondary accent color rather than inverting it like almost everything else.